ssl/tls13_server.cc

```diff
@@ -112,7 +112,13 @@ static int ssl_ext_supported_versions_add_serverhello(SSL_HANDSHAKE *hs,
 
 static const SSL_CIPHER *choose_tls13_cipher(const SSL *ssl) {
   STACK_OF(SSL_CIPHER) *tls13_ciphers = nullptr;
-  if (ssl->ctx->tls13_cipher_list &&
+  // First check config, otherwise fallback to ctx, and otherwise let client
+  // preference dictate cipher selection
```

Comment:
```suggestion
  // First check config, otherwise fallback to ctx, then let the client
  // preference dictate cipher selection if all else fails.
```

Very minor nit: The if/else if statements are checking identical things on a `SSLCipherPreferenceList`, would abstracting the check into a static helper function help?

Reply:
We'd only be abstracting away the null and size check - not sure if that's a valuable "helper". Updated the documentation.